src/actions/actionImport.js

```diff
@@ -10,8 +10,9 @@ export function parseActionImportFile(file) {
 
         reader.onload = e => {
             // TODO: Check type and support CSV as well
-            let tableSet = parseWorkbook(e.target.result);
-            resolve({ tableSet });
+            parseWorkbook(e.target.result)
+                .then(tableSet => resolve({ tableSet }))
```

Comment:
If you passed the bare `tableSet`, not wrapped in a `{ tableSet: tableSet }` literal, could you do without the  actionImport.js change, then?

Reply:
@olleolleolle The `parseWorkbook()` function is also used [elsewhere](https://github.com/zetkin/organize.zetk.in/blob/master/src/actions/importer.js), where it's expected to return an object like this. If I understand your suggestion correctly, that would break (or require more changes). Am I right?

Reply:
My suggestion came from: the _other_ change in this PR, is that a follow-up, un-related fix to this repair?

Reply:
action/actionImport is not used elsewhere to my knowledge, I suppose I could just resolve(tableset), then there's no need to change store/actionImport and it would actually be better from a compatibility standpoint if it was used elsewhere for some odd reason.

Reply:
Thanks for the follow-up questions, Richard. And, I hope this comes across as my trying to be understood, rather than anything else. All this is Good Work.

Reply:
Ah, I get it now. I was confused by the naming i guess. The first use of `tableSet` here, i.e. the argument in the function is not actually the table-set, but an object containing the field `tableSet` I believe? So maybe a better name would be `data` or `workbook` or something, along with the change suggested by Olle?

I'll let you decide @niklasva82 since I might be mistaken. Do you want to update the PR before I review?

Reply:
@richardolsson True regarding the naming. I fixed that as well now.